vcx/libvcx/src/api/vcx.rs

```diff
@@ -95,11 +99,6 @@ fn _finish_init(command_handle: u32, cb: extern fn(xcommand_handle: u32, err: u3
 
     ::utils::threadpool::init();
 
-    match ::utils::libindy::payments::init_payments() {
```

Comment:
It seems that the payment plugin is not being initialized anywhere.  In testing it gets initialized inside ::utils::devsetup::tests::init_plugin() but that function is for testing only.  How is the payment plugin initialized outside of testing?

Reply:
Application using libvcx needs to load the plugin beforehand and call init on the plugin for it to bind with libindy.